tests/rest/RSTcheckBearTest.py

```diff
@@ -17,3 +17,60 @@
                 (rst_syntax_good, python_block_bad, python_block_good),
                 (rst_syntax_bad,),
                 settings={'code_block_language_ignore': 'python'})
+
+good_directive1 = '''
+===
+Foo
+===
+
+.. one::
+    Foo
+'''
+
+good_directive2 = '''
+===
+Bar
+===
+
+.. one::
+    Foo
+
+.. two::
+    Bar
+'''
+
+bad_directive = '''
+===
+Bar
+===
+
+.. three::
+    Hello
+'''
+
+good_role = '''
+Foo
+===
+
+:src:`hello_world.py`
+:RFC:`793`
+'''
+
+bad_role = '''
+Bar
+===
+
+:bad:`345`
+'''
+
```

Comment:
Just to make the code more readable. We can good_directive1, good_role just after bad file and good file example. Than maybe have all the 3 tests together.
code_block_language_ignore
ignoredirective
ignorerole

Reply:
I just thought I'll add my stuff at the end, but yes it is a good idea and if that's what other test files follow in general in coala.